mkosi/__init__.py

```diff
@@ -7663,10 +7679,12 @@ def run_ssh(args: MkosiArgs) -> None:
         cmd += ["-p", f"{args.ssh_port}"]
 
     dev, address = find_address(args)
-    cmd += [f"root@{address}{dev}", *args.cmdline]
+    cmd += [f"root@{address}{dev}"]
+
+    cmd += commands or [*args.cmdline]
```

Comment:
Shouldn't `cmd += commands or args.cmdline` work here as well? I'm curious why we need `[*args.cmdline]`